packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3173,6 +3202,126 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     }
   }
 
+  async seekRow(inputString: string, isBackwards = false): Promise<void> {
+    const {
+      gotoValueSelectedColumnName: selectedColumnName,
+      gotoValueSelectedFilter,
+    } = this.state;
+    const { model } = this.props;
+    if (!model.isSeekRowAvailable) {
+      return;
+    }
+
+    if (inputString === '') {
+      return;
+    }
+
+    const columnIndex = model.getColumnIndexByName(selectedColumnName);
+    if (columnIndex === undefined) {
+      return;
+    }
+
+    const selectedColumn = model.columns[columnIndex];
+
+    let searchFromRow;
+
+    if (this.grid) {
+      ({ selectionEndRow: searchFromRow } = this.grid.state);
+    }
+
+    if (searchFromRow == null) {
+      searchFromRow = 0;
+    }
+
+    const isContains = gotoValueSelectedFilter === FilterType.contains;
+    const isEquals =
+      gotoValueSelectedFilter === FilterType.eq ||
+      gotoValueSelectedFilter === FilterType.eqIgnoreCase;
+
+    try {
+      const columnDataType = TableUtils.getNormalizedType(selectedColumn.type);
+
+      let rowIndex;
+
+      switch (columnDataType) {
+        case TableUtils.dataType.STRING: {
+          rowIndex = await model.seekRow(
+            isBackwards === true ? searchFromRow - 1 : searchFromRow + 1,
+            selectedColumn,
+            'String',
```

Comment:
Instead of passing a raw string here, pass in the `dh.ValueType` constants: https://github.com/deephaven/deephaven-core/blob/89cbd6aafb2b10f349546b91220245fabac74c53/web/client-api/src/main/java/io/deephaven/web/client/api/JsTable.java#L96. You'll need to add to `dh.types`.
Basically anywhere you're calling `model.seekRow`, you should pass a `dh.ValueType` constant in.

See how `VariableType` is defined in dh.types.ts and used in `IrisGridPanel` as an example.